### PR TITLE
Add travis status

### PR DESCRIPTION
**Pull request by zeckalpha**

### CODE REVIEW

File: destalinator.py [hunk 1]
```diff
@@ -81,7 +81,11 @@ def debug(self, message):
 
     def get_earliest_archive_date(self):
         """Return a datetime.date object representing the earliest archive date."""
-        date_string = os.getenv(self.config.get('earliest_archive_date_env_varname')) \
+        date_string = \
+            ( \
+                self.config.get('earliest_archive_date_env_varname') \
+                and os.getenv(self.config.get('earliest_archive_date_env_varname')) \
```
Comment (h3h) [inline destalinator.py]: Why the `and`? `os.getenv()` falls back to returning `None`.

Comment (zeckalpha) [inline destalinator.py]: In 2.7, but in 3.6 it was failing when a non `str` was passed in as the key.

Comment (zeckalpha) [inline destalinator.py]: https://travis-ci.org/randsleadershipslack/destalinator/jobs/240888575

Comment (h3h) [inline destalinator.py]: Oh, well that sucks. Could we set the fallback to the empty string?
```python
date_string = os.getenv(self.config.get('earliest_archive_date_env_varname', '')) \
…
```

Comment (zeckalpha) [inline destalinator.py]: Go for it!

**Pull request merged**